git-extra/PKGBUILD

```diff
@@ -34,13 +34,13 @@ source=('inputrc'
         '99-post-install-cleanup.post'
         'astextplain'
         'git-sdk.sh')
-md5sums=('3fab57079f0322efe256e95fe29d516f'
-         'e7ad03fffc29e619e402dbd5ec9ef74c'
-         'bfb591886b2a28af3334521e71198b74'
```

Comment:
Are you sure about these? Seems to me that you may have checked out the files with CR/LF line endings, when the MD5 checksums apply to the LF-only versions instead.